src/main/java/net/smartcosmos/dao/relationships/impl/RelationshipPersistenceService.java

```diff
@@ -273,11 +328,55 @@ private RelationshipEntity persist(RelationshipEntity relationshipEntity) throws
         }
     }
 
-    private List<RelationshipResponse> convert(List<RelationshipEntity> entityList) {
+    private List<RelationshipResponse> convertList(List<RelationshipEntity> entityList) {
         return entityList.stream()
```

Comment:
This could be a `Converter` in its own right and then the code is cleaner.

`convertList` with type `List<RelationshipEntity>` is somewhat redundant. `convert()` would be better but it's not worth fighting over.